nautobot/dcim/forms.py

```diff
@@ -2077,27 +2077,27 @@ class DeviceFilterForm(
         label="Virtual chassis member",
         widget=StaticSelect2(choices=BOOLEAN_WITH_BLANK_CHOICES),
     )
-    console_ports = forms.NullBooleanField(
+    has_console_ports = forms.NullBooleanField(
```

Comment:
This is potentially a problem because it means that pre-1.3 URL filters won't work, even if this is transparent and only really utilized in the context of UI filtering, it's something to consider.

Reply:
Also impacts REST API filtering, doesn't it?

Reply:
No, since I added filter fields to `DeviceFilterSet` and didn't take any away. So the schema will display more filters with `has_` in their name, but will still work as before.

Reply:
This was reverted.